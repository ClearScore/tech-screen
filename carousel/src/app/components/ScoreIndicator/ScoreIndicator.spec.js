import ScoreIndeicator from './ScoreIndicator';
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('ScoreIndicator', () => {
  const wrapper = shallow(<ScoreIndeicator />);
  it('renders correctly', () => {
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});
