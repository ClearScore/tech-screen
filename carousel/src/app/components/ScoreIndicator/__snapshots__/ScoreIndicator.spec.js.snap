// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScoreIndicator renders correctly 1`] = `
<div
  className="container-circle"
>
  <div
    className="container-text"
  >
    <styled.p>
      Your credit score is
    </styled.p>
    <styled.p />
    <styled.p>
      out of 700
    </styled.p>
  </div>
</div>
`;
